test/unit/foreman/access_permissions_test.rb

```diff
@@ -10,59 +11,8 @@
 #
 # In particular, it's important that actions for AJAX requests are added to
 # an appropriate permission so views using those requests function.
-class AccessPermissionsTest < ActiveSupport::TestCase
-  MAY_SKIP_REQUIRE_LOGIN = [
```

Comment:
Moving the constants were actually the whole reason I opened the PR. 

Without any of these changes, I could still test /my_plugin/ routes easily, however if I add any routes to the existing foreman routes I can't easily choose to test all the foreman routes because I need access to these constants.  'requiring'  the main test file (as it is in master) made those tests to get executed due to the way they were rewritten.  The only options I saw were:

1) Move the constants to some other file that could be required and re-used (which is what is here)
2) Rewrite the tests so that they didn't declare one test per route.
3) Just redefine them in my plugin

1) seemed like the best route to me, but I'm happy to look at other suggestions